modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -1817,6 +1817,136 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12167. On the Product Details page user can see option values with two different SKUs linked"
```

Comment:
User can see SKU updated on the product details page when values are selected from multiple options.

I think this would be a better description